src/sentry/runner/commands/run.py

```diff
@@ -748,6 +754,7 @@ def dev_consumer(consumer_names):
             max_poll_interval_ms=None,
             synchronize_commit_group=None,
             synchronize_commit_log_topic=None,
+            enable_dlq=False,
```

Comment:
How come you're not passing the actual value through?

Reply:
This is the one where they roll all the consumer into a single thread

Reply:
👍